src/modules/pos/validator/components/ChangeCommission/Info/ChangeCommissionInfo.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  useCurrentCommissionPercentage
+} from '@pos/validator/hooks/useCurrentCommissionPercentage';
+import { convertCommissionToPercentage } from '@pos/validator/utils';
+import styles from './ChangeCommissionInfo.css';
+
+export const ChangeCommissionInfo = ({ transactionJSON = {} }) => {
+  const { t } = useTranslation();
+  const { currentCommission } = useCurrentCommissionPercentage();
+
+  return (
+    <>
+      <section className={styles.commission}>
+        <label>{t('Old Commission')} (%)</label>
+        <label className={`${styles.label}`} data-testid="current-commission">{convertCommissionToPercentage(currentCommission)}</label>
```

Comment:
This is already done in the `useCurrentCommissionPercentage` hook. Now doing it here again is resulting in wrong value by 100.
<img width="498" alt="image" src="https://user-images.githubusercontent.com/8784876/218060111-ae8da49c-c24e-480e-9e27-ed7eda5b6526.png">

even though old commission is 100
<img width="506" alt="image" src="https://user-images.githubusercontent.com/8784876/218060202-a85cdce9-02f3-48d1-abff-6ff2c1da4843.png">

Reply:
good catch